pyproject.toml

```diff
@@ -30,13 +30,27 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-tests = ["pytest==8.3.3", "pytest-pyvista==0.1.9", "pytest-cov==6.0.0"]
+pyvistaqt = [
+    "pyside6 >= 6.8.0,<7",
+    "pyvistaqt >= 0.11.1,<1",
+]
+tests = [
+    "pytest==8.3.3",
+    "pytest-pyvista==0.1.9",
+    "pytest-cov==6.0.0",
+    "pyside6==6.7.3",
+    "pyvistaqt==0.11.1,<1",
+    "pytest-qt"
+]
+
 doc = [
     "ansys-sphinx-theme==1.2.0",
     "jupyter_sphinx==0.5.3",
     "jupytext==1.16.4",
     "nbsphinx==0.9.5",
     "numpydoc==1.8.0",
+    "pyside6==6.7.3",
+    "pyvistaqt==0.11.1,<1",
```

Comment:
I would pin it to a specific version if the doc... if needed 😄 or just drop it in that case

Reply:
Not needed for now.. Removing it.